buildpack/runtime.py

```diff
@@ -75,6 +75,22 @@ def stage(buildpack_dir, build_path, cache_path):
         os.path.join(build_path, ".local", "m2ee.yaml"),
     )
 
+    util.mkdir_p(os.path.join(build_path, ".local", "mx-agent"))
```

Comment:
Please don't add this to `runtime.py`, but to the staging phase in `mx-java-agent.py`.

Reply:
Done